test/Acornima.Tests/Acornima.Tests.csproj

```diff
@@ -14,19 +14,19 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="DiffEngine" Version="13.0.2" />
-    <PackageReference Include="GitHubActionsTestLogger" Version="2.3.3" PrivateAssets="all" />
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.9.0" />
-    <PackageReference Include="Newtonsoft.Json" Version="13.0.3" />
-    <PackageReference Include="UnicodeInformation" Version="2.7.1" />
-    <PackageReference Include="xunit" Version="2.7.0" />
-    <PackageReference Include="xunit.analyzers" Version="1.11.0" />
-    <PackageReference Include="xunit.runner.visualstudio" Version="2.5.7" PrivateAssets="all" />
-    <PackageReference Include="xunit.runner.console" Version="2.7.0" PrivateAssets="all" />
+    <PackageReference Include="DiffEngine" VersionOverride="13.0.2" />
```

Comment:
I don't get the VersionOverride here. It's unlikely we'll use a different version of this package in other projects, so couldn't the version be set centrally, in Directory.Packages.props?

Reply:
Reverted to older version globally, they don't support older FW anymore.